### DISCUSSION TITLE
Timeout values with octoprint

### DISCUSSION THREAD

**Discussion by bakaneko59** [Jyers/Marlin#984] [Q&A] [answered]
Could someone who is successfully using Octoprint with Jyers please post the timeout values they're using?  I still get serial errors in the middle of my prints, which then stop printing and I have to start over.  The error messages generally say to adjust my timeout values so I want to see what someone else has had success with.  In the meantime I've gone back to primarily using the SD card to print because Octoprint is so frustrating.  And before someone suggests it, I'm already using a short (18 inch) shielded USB cable with ferrite baluns and 5v pin disconnected).

![image](https://user-images.githubusercontent.com/16111694/121225274-fa766700-c84e-11eb-9361-e213a4eba73a.png)

**Comment by JJCrash** [comment]
I was having those problems and went to the plugins on octoprint and uninstalled all of them. Shut down the pi waited about 5 minutes and went back in and started slowly started to add the plugins again. Have not had that problem since i did this. Not Jyers its a plugin for octoprint.

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Reply by bakaneko59** [reply]
I tried starting in safe mode and got this... 
![20210608_203627](https://user-images.githubusercontent.com/16111694/121279524-4b5f7d00-c89a-11eb-8092-5dd49aaf6dd3.jpg)

**Comment by Jyers** [comment, marked as answer]
Are you running your USB anywhere near your heater wires or power lines? Shielding is good, but it can only do so much.

**Comment by BlaydeRunner08** [comment]
I used to have disconnects between OctoPrint and my printer, even before flashing to Jyers.  My issue turned out to be a faulty USB cable, which is more common than you think!
